src/CommunityToolkit.Maui/Converters/VariableMultiValueConverter.shared.cs

```diff
@@ -66,7 +66,8 @@ public object Convert(object[]? values, Type targetType, object? parameter = nul
 			MultiBindingCondition.Exact => count == Count,
 			MultiBindingCondition.GreaterThan => count > Count,
 			MultiBindingCondition.LessThan => count < Count,
-			_ => count == boolValues.Length,
+			MultiBindingCondition.All => count == boolValues.Length,
+			_ => throw new NotSupportedException($"{ConditionType} not yet supported")
```

Comment:
This doesn't change any logic. It just explicitly defines `MultiBindingCondition.All` and sets the default case to throw an exception. 

This default case will help us catch bugs in the future if/when we add another named constant to `MultiBindingCondition`.